tox.ini

```diff
@@ -1,152 +1,63 @@
 [tox]
-envlist = py27
+envlist = py27-tests
```

Comment:
There's little point in looking at the diff for the `tox.ini` file itself, it's basically a complete rewrite of the entire file.

Reply:
What `envlist` means here, by the way, is the default list of "testenvs" to run if `tox` is run with no `-e` argument. If `-e` is given on the command line it overrides the `envlist` from `tox.ini`/ So `tox` is the same as `tox -e py27-tests`. And `tox -e py36-dev` overrides `envlist` entirely.

You _could_ put multiple things in `envlist` to have `tox` run more than one testenv. E.g. `envlist = py27-tests,py36-tests,py27-functests,...`.